pom.xml

```diff
@@ -169,8 +169,8 @@
         <commons-math.version>2.2</commons-math.version>
         <collections.version>3.2.1</collections.version>
         <scoverage.plugin.version>1.1.1</scoverage.plugin.version>
-        <spark-version.project>1.5-plus</spark-version.project>
-        <spark.version>1.5.1</spark.version>
+        <spark-version.project>1.6</spark-version.project>
+        <spark.version>1.6.0</spark.version>
```

Comment:
why 1.6.0? I remember 1.6.0 netty blockmanager has problem. why not the latest stable version for spark 1.6?